meta-mender-imx/templates/local.conf.append

```diff
@@ -0,0 +1,18 @@
+
+# Appended fragment from meta-mender-community/meta-mender-imx/templates
+IMAGE_CLASSES += "mender-setup-imx"
+
+# General Settings
+MENDER_BOOT_PART_SIZE_MB_imx8mnevk = "256"
```

Comment:
Why so large? I would assume that we actually can disable the boot part completely on this board?

Reply:
Not for GRUB-EFI integration.

Reply:
Also, the optee and M4 bins are by default in /boot.  That may be changeable but I've not investigated yet.

Reply:
> Not for GRUB-EFI integration.

Ah, true.